### DISCUSSION TITLE
Gas-phase fractions from orbitrap

### DISCUSSION THREAD

**Discussion by bshashikadze** [vdemichev/DiaNN#105] [Q&A] [answered]
Hi,
I have gas phase fractions which I would like to use for library generation (in combination with PROSIT predicted library). Gas phase fractions were measured on Qexactive HFx using staggered windows. Is this supported by DIANN? I see something like this is supported for Gas phase fractions from TOF, so I am wondering about measurments from Orbitrap..
Thank you very much!
Bachi

**Comment by vdemichev** [comment, marked as answer]
Hi Bachi,

Yes, it's supported. Although there's no need to use Prosit, DIA-NN has its own built-in deep learning predictor.

Best wishes,

Vadim

**Comment by bshashikadze** [comment]
Hi Vadim,

Amazing!
Thanks a lot for answer!
Bachi
